python/ray/serve/_private/http_proxy.py

```diff
@@ -68,9 +68,18 @@
 if os.environ.get("SERVE_REQUEST_PROCESSING_TIMEOUT_S") is not None:
     logger.warning(
         "The `SERVE_REQUEST_PROCESSING_TIMEOUT_S` environment variable has "
-        "been deprecated. Please use `RAY_SERVE_REQUEST_PROCESSING_TIMEOUT_S` "
-        "instead. `SERVE_REQUEST_PROCESSING_TIMEOUT_S` will be ignored in "
-        "future versions."
+        "been deprecated. Please use `request_timeout_s` in the serve config instead. "
+        "`SERVE_REQUEST_PROCESSING_TIMEOUT_S` will be ignored in future versions. See: "
+        "https://docs.ray.io/en/master/serve/api/doc/ray.serve.schema.HTTPOptionsSchema"
```

Comment:
Before branch cut, we should permalink this URL to Ray 2.6 (or whatever version the user is running). @edoakes is there a better way to do remember to do this than just leaving a `TODO`?

Reply:
Sg, latest release is 2.5.1, will pin to that one for now. It will still be able to link to the doc, just won't anchor to `request_timeout_s`. Looks like this if you wonder https://docs.ray.io/en/releases-2.5.1/serve/api/doc/ray.serve.schema.HTTPOptionsSchema.html#ray.serve.schema.HTTPOptionsSchema.request_timeout_s I think should be fine not to leave a TODO and come back later 🙂